### DISCUSSION TITLE
Programmatically disconnect wallet without wagmi

### DISCUSSION THREAD

**Discussion by jtlemkin** [WalletConnect/walletconnect-monorepo#5374] [Developer Support]
Hi, I was wondering if it was possible to programmatically disconnect a wallet connect wallet from my dapp without needing to install wagmi as a dependency.

**Comment by hugoorlov** [comment]
hopefully this is possible, because I was getting anomaly when disconnecting using wagmi disconnect function

somehow the new Appkit connect modal still showing the balance and wallet address eventhough the wagmi disconnect function has been triggered, when I reload the page, it become normal again and the modal will asking to login our wallet again
